src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2151,7 +2161,9 @@ because xbuild doesn't support framework reference assemblies.
       LibraryProjectJars="@(ExtractedJarImports)"
       TlsProvider="$(AndroidTlsProvider)"
       UncompressedFileExtensions="$(AndroidStoreUncompressedFileExtensions)"
-      ProjectFullPath="$(MSBuildProjectFullPath)">
+      ProjectFullPath="$(MSBuildProjectFullPath)"
+      IncludeWrapSh="$(AndroidIncludeWrapSh)"
```

Comment:
Should `$(AndroidIncludeWrapSh)` be documented in `Documentation/guides/building-apps/build-properties.md`?